cmd/lncli/cmd_query_mc_probability.go

```diff
@@ -0,0 +1,70 @@
+// +build routerrpc
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/btcsuite/btcutil"
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/routing/route"
+	"github.com/urfave/cli"
+)
+
+var queryMcProbCommand = cli.Command{
+	Name:      "querymcprob",
```

Comment:
Wondering is it's necessary to include `mc` in the command name?
Mission control is an internal concept that users might not be familiar with, `queryPaymentProbability` or something more user suited may be more informative?

Reply:
just `queryprob`?

Reply:
changed to `queryprob`